Docs/SSE: Misc updates and corrections (#35882)
grafana/grafana: docs/sources/panels/expressions.md

## Using expressions

> The primary use case for expressions is for use with the new [Grafana 8 alerting]({{< relref "../alerting/unified-alerting/" >}}). Like alerting, processing is done server-side, so expressions can operate without a browser session. However, expressions can be used with backend data sources and visualization as well.

Reviewer: ```suggestion
Expressions are primarily used by the new [Grafana 8 alerts]({{< relref "../alerting/unified-alerting/" >}}). The processing is done server-side, so expressions can operate without a browser session. However, expressions can also be used with backend data sources and visualization.
```
